src/App.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+
 import { News } from "./pages/News";
 
 function App() {
```

Comment:
here should be your search value in the state
you pass onSearchChange function to the Header props
Header passes it into the Search via props